app/assets/stylesheets/pages/_show.scss

```diff
@@ -0,0 +1,8 @@
+h1 {
+  text-align: center;
+}
+
+.row {
+  display: flex;
+  flex-direction: column;
+}
```

Comment:
Row is a bootstrap class so usually we wouldn't wanna mess with the styling they have. We can flex box properties on individual elements with additional bootstrap classes.